### ISSUE TITLE
Keyboard focus is not moving to the Python, C#, java, javascript, c++ tabs: A11y_ONNX Runtime & Ecosystem_Runtime_Keyboard

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/onnxruntime#24997]
#A11yTCS #A11yMAS #A11ySev3 #MAS2.1.1#Keyboard #DesktopWeb #Chrome #Win11 #BM_ONNX Runtime & Ecosystem_Web_June2025; #ONNX Runtime & Ecosystem; #Benchmark #rev:moda
### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/
Chrome Version: 120.0.6099.130
OS: Windows 11 version 23H2
### Steps to Reproduce
Hit the URL "https://onnxruntime.ai/".
"ONNX Runtime" home page appeared.
Tab till "Hardware Acceleration" and tab again.
observe that Keyboard focus is not moving to the Python, C#, java, javascript, c++ tabs.
### Actual:
Keyboard focus is not moving to the Python, C#, java, javascript, c++ tabs
**Note: There is an alternative way to access the code by press enter on the More button**
### Expected
Keyboard focus should be moving to the Python, C#, java, javascript, c++ tabs.
### User Impact:
Keyboard users will face difficulties if the tab controls Python, C#, java, javascript, c++  are not accessible with keyboard.
### Recommendations:
Refer[ Developer Resources (sharepoint.com)](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx).
### MAS References:
[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)
Check out [ MAS 2.1.1: Keyboard Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-2.1.1-Keyboard-Guide.aspx?)
Attachment
[A11y_ONNX Runtime & Ecosystem_Runtime_Keyboard 1.zip](https://github.com/user-attachments/files/20655041/A11y_ONNX.Runtime.Ecosystem_Runtime_Keyboard.1.zip)
